server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -92,9 +91,13 @@ export class TenantManager implements core.ITenantManager {
 				password: generateToken(tenantId, documentId, key, null),
 				user: tenantId,
 			};
-			return {
+			const headers: AxiosRequestHeaders = {
 				Authorization: getAuthorizationTokenFromCredentials(credentials),
 			};
+			if (storageName) {
+				headers.StorageName = storageName;
+			}
+			return headers;
```

Comment:
@znewton , @hedasilv  Are our communication to historian all using this `gitManager`? If so, maybe we can break the contract here to make `storageName` required, and then all the consumer of historian would need to provide the `storageName` to historian, make historian totally free from storageName dependency calculation? @yangg-msft